coupon.go

```diff
@@ -289,20 +291,20 @@ func (cr *CouponRequest) Delete(ctx context.Context, couponCode string) (*Coupon
 }
 
 func (cr *AppliedCouponRequest) GetList(ctx context.Context, appliedCouponListInput *AppliedCouponListInput) (*AppliedCouponResult, *Error) {
-	jsonQueryParams, err := json.Marshal(appliedCouponListInput)
+	urlValues, err := query.Values(appliedCouponListInput)
 	if err != nil {
 		return nil, &Error{Err: err}
 	}
 
-	queryParams := make(map[string]string)
-	if err = json.Unmarshal(jsonQueryParams, &queryParams); err != nil {
-		return nil, &Error{Err: err}
-	}
-
 	clientRequest := &ClientRequest{
-		Path:        "applied_coupons",
-		QueryParams: queryParams,
-		Result:      &AppliedCouponResult{},
+		Path:      "applied_coupons",
+		UrlValues: urlValues,
+		Result: &AppliedCouponResult{
+			Meta: Metadata{
+				CurrentPage: 1,
+				NextPage:    0,
+			},
```

Comment:
Do we need to provide the Meta at this stage? :thinking:

Reply:
I believe this is a left-over from debugging the test. I'll remove it. 👍

Reply:
Done.